packages/language/src/document.js

```diff
@@ -634,11 +704,162 @@ Document.prototype.detectSentiment = function(options, callback) {
     var originalResp = extend(true, {}, resp);
     var sentiment = Document.formatSentiment_(resp.documentSentiment, verbose);
 
+    if (verbose) {
+      sentiment = extend(sentiment, {
+        sentences: Document.formatSentences_(resp.sentences, verbose),
+        language: resp.language
+      });
+    }
+
     callback(null, sentiment, originalResp);
   });
 };
 
 /**
+ * Detect syntax from the document.
+ *
+ * @resource [documents.analyzeSyntax API Documentation]{@link https://cloud.google.com/natural-language/reference/rest/v1/documents/analyzeSyntax}
+ *
+ * @param {object=} options - Configuration object. See
+ *     [documents.annotateText](https://cloud.google.com/natural-language/reference/rest/v1/documents/analyzeSyntax#request-body).
+ * @param {boolean} options.verbose - Enable verbose mode for more detailed
+ *     results. Default: `false`
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error occurred while making this request.
+ * @param {object} callback.syntax - The recognizd syntax from the text.
+ * @param {string[]} callback.syntax.sentences - Sentences detected from the
+ *     text.
+ * @param {object[]} callback.syntax.tokens - Parts of speech that were
+ *     detected from the text.
+ * @param {string} callback.syntax.language - The language detected from the
+ *     text.
+ * @param {object} callback.apiResponse - The full API response.
+ *
+ * @example
+ * document.detectSyntax(function(err, syntax) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   // syntax = {
+ *   //   sentences: [
+ *   //     'Google is an American multinational technology company ' +
+ *   //     'specializing in Internet-related services and products.'
+ *   //   ],
+ *   //   tokens: [
+ *   //     {
+ *   //       text: 'Google',
+ *   //       partOfSpeech: 'Noun (common and proper)'
```

Comment:
Let's keep the `tag` property here as well in case the user needs our map to help them work with the data.